src/goto-programs/goto_convert_functions.cpp

```diff
@@ -41,6 +41,9 @@ void goto_convert_functionst::goto_convert()
 
   forall_symbols(it, symbol_table.symbols)
   {
+    // For each code symbol that:
```

Comment:
This comment probably doesn't help and isn't related to this code change - remove from PR